### DISCUSSION TITLE
Revise the mobile view

### DISCUSSION THREAD

**Discussion by HMarzban** [docs-plus/docs.plus#105] [Ideas] [answered]
I like the user experience of Google Docs! It gives a more mature feeling!
We can have these features on our side.

@edsaperia What do you think? 
(I suggest having these features for the beta release)

![image](https://user-images.githubusercontent.com/20157508/235413412-85c5fc6d-ed8b-4e78-b925-3312ab22b675.png)
------
![image](https://user-images.githubusercontent.com/20157508/235413540-3522539f-5058-49c7-95f9-e0c54dc4ffaa.png)
------
![image](https://user-images.githubusercontent.com/20157508/235413810-e8693b83-60dd-4d41-89ab-34ee8b752bab.png)

**Comment by edsaperia** [comment]
Can you be more specific about what you're suggesting compared to what we have now? They seem very similar to me. Is it just that in google docs the top bar hides when you scroll down?

**Reply by HMarzban** [reply]
Yes, one of the features we can have is to indicate to scroll down; when a user scrolls down for more concentration on the content, we can hide the Big blue button(7) and the title bar(6).

Another one is in edit mode. When a user enters the edit mode, It would be nice to have a toolbar with essential functionality at the top of the page(3), Like undoing and redoing, etc.

**Reply by edsaperia** [reply, marked as answer]
Perhaps these are nice features but I don't think they are very important now, since the current experience is totally fine, and the mobile experience will change in the future.

**Reply by HMarzban** [reply]
Noted.
